hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieROTablePathFilter.java

```diff
@@ -202,10 +212,6 @@ public boolean accept(Path path) {
           }
           nonHoodiePathCache.add(folder.toString());
           return true;
-        } finally {
-          if (fsView != null) {
-            fsView.close();
-          }
```

Comment:
How should the cached views been closed ?

Reply:
`fsView.close()` will do 
```
  @Override
  public void close() {
    closed = true;
    super.reset();
    partitionToFileGroupsMap = null;
    fgIdToPendingCompaction = null;
    fgIdToBootstrapBaseFile = null;
    fgIdToReplaceInstants = null;
  }

because of we recycling fsView, we can't close it. Although we create fsView for each baseDir. And will cause no memory leak maybe.
```